### PR TITLE
Make Jetpack Protect timeout filterable and increase default to 30 seconds

### PR DESCRIPTION
**Pull request by samhotchkiss**

### CODE REVIEW

File: modules/protect.php [hunk 1]
```diff
@@ -666,11 +666,24 @@ function protect_call( $action = 'check_ip', $request = array () ) {
...
+		/**
+		 * Filter controls maximum timeout in waiting for reponse from protect servers
+		 *
+		 * @module protect
+		 *
+		 * @since 4.1.0
+		 *
+		 * @param int $timeout Max time (in seconds) to wait for a response
+		 */
+
```
Comment (jeherve) [inline modules/protect.php]: You'll need to remove the space for the parser to detect the inline docs.

File: modules/protect.php [hunk 2]
```diff
@@ -666,11 +666,24 @@ function protect_call( $action = 'check_ip', $request = array () ) {
 			$request['multisite'] = get_blog_count();
 		}
 
+
+		/**
+		 * Filter controls maximum timeout in waiting for reponse from protect servers
```
Comment (jeherve) [inline modules/protect.php]: In the past, we've tried to add periods at the end of each sentence in filter descriptions and param descriptions.

**Comment by jeherve** [comment]
I've made the inline doc changes, and also added an extra check to make sure the filtered data is a non-negative integer. Should be good to merge now!

**Comment by kraftbj** [comment]
This is milestoned at 4.0.4. If it goes in then, need to update docblock to `@since`. I'm 👍 to this and to it being in 4.0.4.

**Comment by jeherve** [comment]
@kraftbj Good call. I updated it. 👍

**Pull request merged**